src/deluge/gui/views/automation_view.cpp

```diff
@@ -3697,21 +3697,10 @@ ActionResult AutomationView::verticalEncoderAction(int32_t offset, bool inCardRo
 		else if (!currentUIMode && outputType != OutputType::KIT) {
 			actionLogger.deleteAllLogs();
 
-			offset = std::min((int32_t)1, std::max((int32_t)-1, offset));
+			clip->nudgeNotesVertically(
+			    offset, Buttons::isShiftButtonPressed() ? VerticalNudgeType::ROW : VerticalNudgeType::OCTAVE,
```

Comment:
nit but can we put the nudge type in a variable so this call isn't split on 3 lines?